bibliopixel/drivers/webvis/__init__.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# encoding: utf-8
+from .. driver_base import DriverBase, ChannelOrder
+
+import logging
+import signal
+import sys
+import time
+import threading
+import uuid
+from ... import log
+
+from . SimpleWebSocketServer import WebSocket, SimpleWebSocketServer
+
+
+class VisWebSocket(WebSocket):
+
+    def __init__(self, *args, **kwargs):
+        super(VisWebSocket, self).__init__(*args, **kwargs)
+        self.kinect = None
+        self.oid = None
+        self.connected = False
+
+    def handleConnected(self):
+        log.debug(self.address, 'connected')
+        self.connected = True
+        # self.kinect = KinectFactory.create_kinect()
+        # self.oid = uuid.uuid1()
+        # self.kinect.add_observer(self.oid, self.send_depth)
+
+    def handleClose(self):
+        # self.kinect.remove_observer(self.oid)
+        self.connected = False
+        log.debug(self.address, 'closed')
+
+    def handleMessage(self):
+        # degs = int(float(self.data))
+        # self.kinect.set_tilt(degs)
+        pass
+
+    def send_pixels(self, pixels):
+        if self.connected:
+            self.sendMessage(pixels)
+
+
+class ws_thread(threading.Thread):
```

Comment:
There's never a good reason to subclass thread!  

You should (almost) always pass a callable to a constructor of Thread.